### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by JaredHook**

### CODE REVIEW

File: angular-techDraft/src/app/about/about.component.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<section>
+    <div class="header">
+        <a class="active" href="#"><button aria-label=""
+            class="h-jI8r2f9ozTNqu_2TBeY" role="navigation">
+
```
Comment (JuliaDunayeva) [inline angular-techDraft/src/app/about/about.component.html:5]: Please remove the empty line

File: angular-techDraft/src/app/about/about.component.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<section>
+    <div class="header">
+        <a class="active" href="#"><button aria-label=""
+            class="h-jI8r2f9ozTNqu_2TBeY" role="navigation">
+            
+            <span class="_1GieMuLljOrqnVpRAwz7VP"><strong> <h1>Your profile</h1></strong></span>
+            <img src="https://www.redditstatic.com/avatars/avatar_default_05_FFD635.png"
+                class="_165RkdCO9QoaErMgkkAsEc _1jKtP65jnxLmoGCSqxAgkZ">
+
```
Comment (JuliaDunayeva) [inline angular-techDraft/src/app/about/about.component.html:9]: Please remove the empty line

File: angular-techDraft/src/app/about/about.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (JuliaDunayeva) [inline angular-techDraft/src/app/about/about.component.spec.ts]: please remove the empty line

File: angular-techDraft/src/app/login/login.component.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
+
```
Comment (JuliaDunayeva) [inline angular-techDraft/src/app/login/login.component.css:1]: Why do we need extra-lines at the beginning of the file?

File: angular-techDraft/src/app/testComponent/test.component.css [hunk 5]
```diff
@@ -0,0 +1,17 @@
+/*.ui.container {
```
Comment (JuliaDunayeva) [inline angular-techDraft/src/app/testComponent/test.component.css:1]: What is this component? It should be removed

File: angular-techDraft/src/index.html [hunk 6]
```diff
@@ -1,15 +1,14 @@
...
-    <link rel="icon" type="image/x-icon" href="favicon.ico">
-  
-  </head>
+<head>
+  <meta charset="utf-8">
+  <title>Class Project front page edit</title>
+  <base href="/">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css">
```
Comment (JuliaDunayeva) [inline angular-techDraft/src/index.html:9]: Why do we link semantic here?

**Review by bouteina-kadiri — APPROVED** [review]

**Pull request closed without merging**